### PR TITLE
Regular maintenance

### PR DESCRIPTION
**Pull request by application-developer-DA**

### CODE REVIEW

File: rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# This project uses rustfmt to format source code. Run `cargo +nightly fmt [-- --check].
+# https://github.com/rust-lang/rustfmt/blob/master/Configurations.md
+
+
```
Comment (strohel) [inline rustfmt.toml:4]: nit: I'd remove this second blank line

**Review by strohel — APPROVED** [review]
Nice, looks reasonable. One super-tiny optional nit.

**Comment by agalakhov** [comment]
Looks good. I would also update all dependencies.

**Comment by application-developer-DA** [comment]
@agalakhov , agree, this is in my pipeline as well: we have for instance a PR from `dependabot`: https://github.com/snapview/tungstenite-rs/pull/153

**Review by jxs — APPROVED** [review]

**Pull request merged**